src/main/java/com/example/sixthmafiabot/validators/PlayerValidator.java

```diff
@@ -20,21 +19,12 @@
 public class PlayerValidator extends BaseValidator {
 
     @Autowired
-    PlayerRepository playerRepository;
+    SpringPlayerRepository playerRepository;
```

Comment:
do not use spring repos in services/validators

Reply:
+